analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -115,7 +113,10 @@ class NuGetSupport(serviceIndexUrls: List<String> = listOf(DEFAULT_SERVICE_INDEX
 
     private val serviceIndices = runBlocking {
         serviceIndexUrls.map {
-            async { mapFromUrl<ServiceIndex>(JSON_MAPPER, it) }
+            async {
+                val json = client.downloadText(it).getOrThrow()
+                JSON_MAPPER.readValue<ServiceIndex>(json)
```

Comment:
These two lines are repeated several times, I think refactoring `mapFromUrl` to contain those two lines and have a better name would be better, for example `ObjectMapper.downloadValue(url: String): T`.

Reply:
Done using a different function name.